language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/rename/RenameTestUtil.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
This should be the year of the first commit. aka. 2021

Reply:
Got it. Updated. Thanks!